docs/pages/getting-started/providers/sailpoint.mdx

```diff
@@ -0,0 +1,255 @@
+
+## Resources
+
+- [SailPoint Identity Secure Cloud Authentication](https://developer.sailpoint.com/docs/api/authentication#choose-authorization-grant-flow)
+- [Managing API Keys and Personal Access Tokens](https://documentation.sailpoint.com/saas/help/common/api_keys.html?h=oauth+client#creating-an-api-key)
+- [SailPoint Developer Community](https://developer.sailpoint.com/discuss/)
+
+## Setup
+
+### Callback URL
+
+<Code>
+  <Code.Next>
+
+```bash
+https://example.com/api/auth/callback/identitySecureCloud
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```bash
+https://example.com/api/auth/callback/identitySecureCloud
+```
+
+  </Code.Svelte>
+</Code>
+
+### Create OAuth Client
+
+Find your Identity Secure Cloud Tenant OAuth Information (can be retrieved under https://{tenant}.api.identitynow.com/oauth/info). Create OAuth Client ([guide](https://documentation.sailpoint.com/saas/help/common/api_keys.html?h=oauth+client#creating-an-api-key))with grant types: AUTHORIZATION_TOKEN and REFRESH_TOKEN, Redirect URL: Callback URL above,   select scopes: sp:scope:all. Note down the clientId and clientSecret.
+
+### Environment Variables
+
+```
+ISC_BASE_API_URL=https://{tenant}.api.identitynow.com
+ISC_BASE_URL=https://{tenant}.identitynow.com
+ISC_CLIENT_ID=
+ISC_CLIENT_SECRET=
+```
+
+### Configuration
+
+<Code>
+  <Code.Next>
+
+```ts filename="/auth.ts"
+import NextAuth from "next-auth"
+
+export const { handlers, auth, signIn, signOut } = NextAuth({
+  providers: [
+    {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: process.env.ISC_CLIENT_ID!,
+      clientSecret: process.env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${process.env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${process.env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${process.env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
+        return {
+            tenant: profile.tenant,
+            id: profile.id,
+            uid: profile.uid,
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+  ],
+})
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```ts filename="/src/auth.ts"
+import { SvelteKitAuth } from "@auth/sveltekit"
+import SailPoint from "@auth/sveltekit/providers/sailpoint"
+
+export const { handle, signIn, signOut } = SvelteKitAuth({
+  providers: [
+    {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: process.env.ISC_CLIENT_ID!,
```

Comment:
The sveltekit environment variables work a bit different, check out this example:

![image](https://github.com/nextauthjs/next-auth/assets/7415984/77d18b83-e358-4ead-876b-8335ff1c8a03)

Reply:
Hello, 

Thanks a lot for the correction. Just made the change in the PR.